### PR TITLE
supply the prompt consent to ensure refresh token comes back from OIDC

### PR DESCRIPTION
**Pull request by floydprice**
### Context

Token and Refresh token is not being returned

### Changes proposed in this pull request

Supply prompt consent 
remove redundant call to super in callback_phase

### Guidance to review

verify access_token contains token and refresh_token

### CODE REVIEW

File: config/initializers/omniauth.rb [hunk 1]
```diff
@@ -17,7 +30,7 @@ def callback_phase
           client.redirect_uri = redirect_uri
           client.authorization_code = authorization_code
           access_token
-          super
+
```
Comment (dankmitchell) [inline config/initializers/omniauth.rb]: @floydprice was this removal intentional? I get a exception on the callback with this removed. An empty array.

**Comment by defong** [comment]
[Manage courses frontend - Sign in refresh token](https://trello.com/c/5tf4hLBw/803-manage-courses-frontend-sign-in-refresh-token)

**Comment by defong** [comment]
[Manage courses frontend - Sign in refresh token](https://trello.com/c/5tf4hLBw/803-manage-courses-frontend-sign-in-refresh-token)
